The-PropertyEstimator-format.md

```diff
@@ -0,0 +1,290 @@
+* A `PhysicalProperty` is the physical property that was measured
+* A `MeasurementMethod` specifying the kind of measurement that was performed
+
+An example of each:
+* `Mixture`: a 0.8 mole fraction mixture of ethanol and water
+* `ThermodynamicState`: 298 kelvin, 1 atmosphere
+* `PhysicalProperty`: mass density
+* `MeasurementMethod`: vibrating tube method
+
+#### Physical substances
+
+We generally use the concept of a liquid or gas `Mixture`, which is a subclass of `Substance`.
+
+A simple liquid has only one component:
+```python
+liquid = Mixture()
+liquid.add_component('water')
+```
+
+A binary mixture has two components:
+```python
+binary_mixture = Mixture()
+binary_mixture.add_component('water', mole_fraction=0.2)
+binary_mixture.add_component('methanol') # assumed to be rest of mixture if no mole_fraction specified
+```
+
+A ternary mixture has three components:
+```python
+ternary_mixture = Mixture()
+ternary_mixture.add_component('ethanol', mole_fraction=0.2)
+ternary_mixture.add_component('methanol', mole_fraction=0.2)
+ternary_mixture.add_component('water')
+```
+
+The infinite dilution of one solute within a solvent or mixture is also specified as a `Mixture`, where the solute has zero mole fraction:
+```python
+infinite_dilution = Mixture()
+infinite_dilution.add_component('phenol', impurity=True) # infinite dilution; one copy only of the impurity
+infinite_dilution.add_component('water')
+```
+
+You can iterate over the components in a mixture:
+```python
+for component in mixture.components:
+    print (component.iupac_name, component.mole_fraction)
+```
+retrieve a component by name:
+```python
+component = mixture.components['ethanol']
+```
+or get the number of components in a mixture:
+```python
+ncomponents = mixture.ncomponents
+```
+or check if a component is an impurity:
+```python
+if component.impurity == True:
+    ...
+```
+
+#### Thermodynamic states
+
+A `ThermodynamicState` specifies a combination of thermodynamic parameters (e.g. temperature, pressure) at which a 
+measurement is performed.
+```python
+from simtk import unit
+thermodynamic_state = ThermodynamicState(pressure=500*unit.kilopascals, temperature=298.15*unit.kelvin)
+```
+We use the `simtk.unit` unit system from [OpenMM](http://openmm.org) for units (though we may later migrate to [`pint`](https://pint.readthedocs.io) for portability).
+
+#### Measurement methods
+
+A `MeasurementMethod` subclass has information specific to the particular method used to measure a property 
+(such as experimental uncertainty guidance).
+
+Some examples:
+* `FlowCalorimetry` for `HeatCapacity` or `ExcessMolarEnthalpy`
+* `VibratingTubeMethod` for `MassDensity`
+
+#### Physical property measurements
+
+A `MeasuredPhysicalProperty` is a combination of `Substance`, `ThermodynamicState`, and a unit-bearing measured 
+property `value` and `uncertainty`:
+```python
+# Define mixture
+mixture = Mixture()
+mixture.addComponent('water', mole_fraction=0.2)
```

Comment:
We've been trying to stick to [PEP8](https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions) here, so we should change CapWords methods to snake_case.